ver5_bokhop:
  hardware_interface:
    joints:
      - ee_joint
      - joint_1
      - joint_2
      - joint_3
      - joint_4

  joint_state_controller:
    type: "joint_state_controller/JointStateController"
    publish_rate: 100

  upper_arm_controller:
    type: "position_controllers/JointTrajectoryController"
    joints: 
      - ee_joint
      - joint_1
      - joint_2
      - joint_3
      - joint_4
